tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -25,107 +22,264 @@ Create VCH - custom base disk
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
+Create VCH - URL without user and password
+    Set Test Environment Variables
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
```

Comment:
@mhagen-vmware Kind of understand why we have these two here, but is it really necessary to clean up dangling vms in each test case?